src/components/AddPun/AddPun.css

```
@@ -0,0 +1,35 @@
+.add-pun__title{
```

```suggestion
pun__title {
```
I think you can set this to just `pun__title`. Also, remember to give some breathing room between the class name and `{`